pkg/controller/migmigration/backup.go

```diff
@@ -306,16 +308,27 @@ func (t *Task) hasBackupCompleted(backup *velero.Backup) (bool, []string) {
 			backup.Namespace,
 			backup.Name)
 		reasons = append(reasons, message)
+		itemsBackedUp, totalItems := getBackupStats(backup)
+		message = fmt.Sprintf(
+			"%s %d out of estimated total of %d objects backed up%s",
```

Comment:
Maybe dumb question, do we need a space at the end for %s and the duration? Or is that taken into account?

Reply:
https://github.com/konveyor/mig-controller/blob/4e103fd65be2c2babf7974172647d3674385c817/pkg/controller/migmigration/backup.go#L190

Answered my own question.